docs/team.rst

```diff
@@ -2,134 +2,118 @@ Read the Docs Team
-
-* `Eric Holscher`_ (Pacific Time)
-* `Anthony Johnson`_ (Mountain Time)
-* `Manuel Kaufmann`_ (Central Time)
-* Your Name Here
-
-**Please don't email us personally for support on Read the Docs.** You can email support@readthedocs.org for any issues you may have. 
+Today we:
 
-Joining the support team
-~~~~~~~~~~~~~~~~~~~~~~~~
+* Serve over **55 million pages** of documentation a month
+* Serve over **40 TB** of documentation a month
+* Host over **80,000 open source projects** and support over **100,000 users**
 
-The best place to start would be to start addressing some of the issues in our issue tracker.
-We have our support policies quite well documented in our :doc:`/contribute`.
-**Be bold**.
-Start trying to reproduce issues that people have,
-or talk to them to get more information.
-After you get the hang of things,
-we'll happily give you the ability to tag and close issues by joining our Support Team.
-
-
-Operations Team
----------------
-
-readthedocs.org is a service that millions of people depend on each month.
-As part of operating the site,
-we maintain a 24/7 on-call rotation.
-This means that folks have to be available and have their phone in service.
-
-Ops team members
-~~~~~~~~~~~~~~~~
-
-* `Eric Holscher`_ (Pacific Time)
-* `Anthony Johnson`_ (Mountain Time)
-* `Matt Robenolt`_ (Pacific Time)
-* Your Name Here
-
-Feel free to ask any of us if you have questions or want to join!
-
-Joining the ops team
-~~~~~~~~~~~~~~~~~~~~
-
-We are always looking for more people to share the on-call responsibility.
-If you are on-call for your job already,
-we'd love to piggy back on that duty as well.
+Read the Docs is provided as a free service to the open source community,
+and we hope to maintain a reliable and stable hosting platform for years to come.
 
-You can email us at dev@readthedocs.org if you want to join our operations team.
-Because of the sensitive nature (API tokens, secret keys, SSL certs, etc.) of the work,
-we keep a private GitHub repository with the operations code & documentation.
+Staff
+-----
 
-The tools that we use are:
+The members of the Staff work full time on the service,
+and we are also honored to have several :ref:`external contributors <major-contributors>`.
 
-* Salt
-* Nagios
-* Graphite/Grafana
-* Nginx
-* Postgres
-* Django
-* Celery
+We mainly fund our operations through :doc:`advertising </advertising/index>`
+and corporate-hosted documentation with `Read the Docs for Business <https://readthedocs.com/>`_,
+and we are supported by a number of generous :doc:`sponsors </sponsors>`.
 
-It's fine if you aren't familiar with all of these things,
-but are willing to help with part of it!
+.. rst-class:: team-roster
 
-**Please reach out if you want to share the on-call responsibility**.
-It really is an important job,
-and we'd love to have it be more geographically distributed.
+============================ ============================
+|eric| Eric Holscher         |anthony| Anthony Johnson
```

Comment:
Is there a reason our 2 names appear bolded? Not sure if that's just a rendering weirdness, or some markup I don't see.

Reply:
The heading of the table I think? This is what the developer tools say:

```
.rst-content table.docutils thead th,
.rst-content table.field-list thead th,
.wy-table thead th {
 font-weight:700;
 border-bottom:2px solid #e1e4e5
}
```

I noticed it but considered it a nice "accident" since you're both in "all teams", founders, and c-level :)